external-crates/move/crates/move-compiler/tests/move_2024/parser/invalid_macro_locs.move

```diff
@@ -0,0 +1,73 @@
+module a::m {
+
+    public struct S<T> { u: T }
+
+    public macro fun make_s<$T>($u: T): S<T> {
+        S{ u: $u }
+    }
+}
+
+module a::m {
+
+    fun valid(): a::m::S<u64> {
+        a::m::make_s!<u64>(0u64)
```

Comment:
Nit, I try to avoid mixing valid and invalid examples in tests. Personally, I find it hard to to then grok the `.exp` file

Reply:
Removed